### PR TITLE
Policy application to files screen test steps and feature issues-18

### PR DESCRIPTION
**Pull request by jaind1046**
Policy application to files screen test steps and feature

### CODE REVIEW

File: specs/step_definitions/content-management-policy-application-to-files.steps.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const {
```
Comment (NadHodeGW) [inline specs/step_definitions/content-management-policy-application-to-files.steps.js:1]: This test will fail, there is no login step

File: specs/step_definitions/content-management-policy-application-to-files.steps.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    I,
+    policyPage,
+    homePage
+} = inject();
+
+Given(/^I have applied the (.*) content management policy to (.*)$/, (flagType, fileType) => {
+    homePage.clickPolicy()
+    policyPage.setFlagTypeForGivenContentFlagsForGivenDocType("embeddedFiles", fileType, flagType)
```
Comment (NadHodeGW) [inline specs/step_definitions/content-management-policy-application-to-files.steps.js:9]: embeddedFiles is used as data but not in the scenario table

File: specs/step_definitions/content-management-policy-application-to-files.steps.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    policyPage.clickSaveChanges()
+});
+
+// TODO implement when functionality available
+When(/^I process (.*)$/, (fileType) => {
+});
+
+// TODO implement when functionality available
```
Comment (NadHodeGW) [inline specs/step_definitions/content-management-policy-application-to-files.steps.js:17]: the file hashes for the original file and the rebuilt will both need to be provided in the scenario table for assertion

**Review by NadHodeGW — CHANGES_REQUESTED** [review]
Please see comments in step file

File: specs/step_definitions/content-management-policy-application-to-files.steps.js (hunk 3 above)
Comment (jaind1046) [reply specs/step_definitions/content-management-policy-application-to-files.steps.js:17]: Deleting this PR to raise new which has all changes according to the comments also.

**Comment by jaind1046** [comment]
rasing new one.

**Pull request closed without merging**